QDS Crafting recipe with Oredictionary Wooden Planks

JasonMcRay opened the pull request:
Make QDS Crafting recipe accept any wooden plank under plankWood oredictionary.

JasonMcRay committed:
QDS Crafting with Oredictionaried Wooden Planks

JasonMcRay committed:
Forgot an import

airbreather commented:
While you're at it, why not `Blocks.GLASS` --> `"blockGlassColorless"` as well, so it'll work with things like that Clear Glass item that Thermal Expansion added back in the day?

JasonMcRay committed:
Oredictionaried also the Glass

So it is compatible with other versions of Normal Glass (like Thermal Expansion Clear Glass)

JasonMcRay commented:
Haven't played with Thermal Expansion so I had no idea. Thanks. I have added it.

canitzp commented:
This pr is already implemented in 1.11.2
